st2reactor/tests/unit/test_sensor_service.py

```diff
@@ -36,8 +36,14 @@ def side_effect(trigger, payload, trace_context):
         self.sensor_service._dispatcher.dispatch = mock.MagicMock(side_effect=side_effect)
         self._dispatched_count = 0
 
-        # reset default configuration value
-        cfg.CONF.system.validate_trigger_payload = False
+        # Previously, cfg.CONF.system.validate_trigger_payload was set to False explicitly
+        # here. Instead, we store original value so that the default is used, and if unit
+        # test modifies this, we can set it to what it was (preserve test atomicity)
+        self.validate_trigger_payload = cfg.CONF.system.validate_trigger_payload
```

Comment:
If they want, yeah. The idea here is to preserve the original/default value for `cfg.CONF.system.validate_trigger_payload` so that each unit test can do whatever they want with it. So, a unit test can change the value of `cfg.CONF.system.validate_trigger_payload` before testing (as many of them already do) or they can not, and use the default value (as my added test does).